src/modules/fundingManager/bondingCurveFundingManager/ToposBondingCurveFundingManager.sol

```diff
@@ -132,13 +134,21 @@ contract ToposBondingCurveFundingManager is
         liquidityPool = ILiquidityPool(_liquidityPool);
         // Set formula contract
         formula = IToposFormula(bondingCurveProperties.formula);
+        // Set capital required
         _setCapitalRequired(bondingCurveProperties.capitalRequired);
+        // Set current seize to MAX_SEIZE
+        _setSeize(bondingCurveProperties.seize);
+        // Set initial
+        _setBaseMultiplier(bondingCurveProperties.basePriceMultiplier);
         // Set sell fee to Max fee at initiation
-        _setSellFee(MAX_SELL_FEE);
+        _setSellFee(bondingCurveProperties.sellFee);
+        // Set buying functionality to open if true. By default buying is false
+        if (bondingCurveProperties.buyIsOpen == true) _openBuy();
+        // Set selling functionality to open if true. By default selling is false
+        if (bondingCurveProperties.sellIsOpen == true) _openSell();
```

Comment:
Just caught my eye that you're using `bondingCurveProperties.buyIsOpen == true` instead of just `bondingCurveProperties.buyIsOpen` which does the same. Do you just do this for simplicity in terms of reading? Was just curious as I think the Bonding Curve contracts are the only ones that do this afaik. Not important and doesn't necessarily need a change, just out of curiosity. It compiles to the same code at least so no difference functionally.

Reply:
Yeah I tend to do this for clarity. I do always try to opt for readability as you've prob have seen in the tests as well :))

However a question back to you would be if there are any negative gas associations to this, in which case I would def change it

Reply:
No gas implications, as it both compiles to the assembly code, so all good. Ideally, I'd love to have the codebase be consistent, which it currently isn't, but let's just keep this like it is and discuss this at a different time with the rest briefly to decide on a general level, I'd say.